Sources/MapboxNavigation/SpeedIndicationView.swift

```diff
@@ -0,0 +1,182 @@
+import UIKit
+import MapboxDirections
+
+/**
+ A view that displays the current speed.
+ */
+public class SpeedIndicationView: UIView {
```

Comment:
@ShanMa1991, as it was stated in https://github.com/mapbox/navigation-sdks/issues/1423 we should not expose new entity to show current speed (like this `UIView`). `SpeedLimitView` itself should be updated to provide the ability to listen to current speed changes.

Then in https://github.com/mapbox/navigation-sdks/issues/1481 @jill-cardamon will use data that your implementation provides to draw it based on the new style.

Reply:
I see. So we should only add the ability to listen to current speed and check whether it exceeds the speed limit in `SpeedLimitView` and also add the Notification of current speed to it, instead of creating the `SpeedIndicationView`. I'll remove the `UIView` and move its function to the `SpeedLimitView` instead.

Reply:
That's right. It'd be great to implement it in a way so that it has well defined APIs other users can consume as well.